includes/Admin/Cross_Origin_Isolation.php

```diff
@@ -216,13 +216,13 @@ public function get_avatar( $avatar, $id_or_email, $size, $default, $alt, array
 	 *
 	 * @since 1.6.0
 	 *
-	 * @param string $html HTML string.
-	 * @param string $attribute Attribute to check for.
-	 * @param string $url URL.
+	 * @param string|mixed $html HTML string.
+	 * @param string       $attribute Attribute to check for.
+	 * @param string       $url URL.
 	 *
-	 * @return string
+	 * @return string|mixed
 	 */
-	protected function add_attribute( string $html, string $attribute, string $url ): string {
+	protected function add_attribute( $html, string $attribute, string $url ) {
```

Comment:
Does `$html` need to be cast to a string here?

Since `$new_html` is always a string, the return type will always be a string, so we could actually retain the return type.

Reply:
Yes, but $new_html is the only thing that can be returned here 

```
if ( $this->starts_with( $url, $site_url ) ) {
			return $html;
		}
		if ( $this->starts_with( $url, '/' ) ) {
			return $html;
		}
```

Reply:
Argh, missed that. Nevermind then.